src/Math-Matrix/PMMatrix.class.st

```diff
@@ -824,6 +824,20 @@ PMMatrix >> qrFactorizationWithPivoting [
 	^ Array with: q with: r with: pivot
 ]
 
+{ #category : #operation }
+PMMatrix >> raisedTo: anInteger [
+
+	" Answers the receiver raised to a power anInteger.
+	 If anInteger is negative, inverse of the receiver is raised to the absolute value of anInteger."
+	self assert: self numberOfRows = self numberOfColumns description: 'Matrix should be square'.
```

Comment:
Check there isn't an `isSquare` method already. Use it if there is one or use the extract method refactor. Add any tests you deem necessary.

Reply:
Yes, `isSquare` method is there, thank you!
I am now using this method.